tests.py

```diff
@@ -11,3 +18,24 @@ def test_render():
     c = Context()
     t = Template('{% extends "admin:admin/base.html" %}')
     t.render(c)
+
+
+overrides = {}
+if django.VERSION < (1, 8):
+    overrides['TEMPLATE_LOADERS'] = [
+        ('django.template.loaders.cached.Loader', settings.TEMPLATE_LOADERS)
+    ]
+else:
+    TEMPLATES = deepcopy(settings.TEMPLATES)
+    opt = TEMPLATES[0]['OPTIONS']
+    opt['loaders'] = [
+        ('django.template.loaders.cached.Loader', opt['loaders'])
+    ]
+    overrides['TEMPLATES'] = TEMPLATES
```

Comment:
Can you move this code block up, before all the `def`s, please? So, we have a continuous list of `test_*` functions. Which reads nicely.